pkg/types/nutanix/platform.go

```diff
@@ -0,0 +1,46 @@
+package nutanix
+
+// Platform stores any global configuration used for Nutanix platforms.
+type Platform struct {
+	// PrismCentral is the domain name or IP address of the Prism Central.
+	PrismCentral string `json:"prismCentral"`
+
+	// Port is the port to use to connect to the Prism Central.
+	Port string `json:"port"`
+
+	// Username is the name of the user to use to connect to the Prism Central.
+	Username string `json:"username"`
+
+	// Password is the password for the user to use to connect to the Prism Central.
+	Password string `json:"password"`
+
+	// PrismElement is the name of the Prism Element cluster to use in the Prism Central.
```

Comment:
Is this a name or a UUID?
```suggestion
	// PrismElementUUID is the UUID of the Prism Element cluster to use in the Prism Central.
```

Reply:
Good catch!